### DISCUSSION TITLE
2015/02/07/where-was-beacon-52

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#18] [Announcements]
# 2015/02/07/where-was-beacon-52

http://localhost:4000/2015/02/07/where-was-beacon-52

**Comment by jengalas** [comment]
Posted  by Steve Bennett
June 4, 2015 at 8:17 pm
_Imported from WordPress_

Zhanna,

Yes this is near enough that I can search on foot.
I wanted to do a search before the leaves and briers came out but with the short days in winter/spring it was dark by the time I got off work.
Now the days are longer I need to make a trip up there after work one day.
I’ll report back when I do.
-Steve

**Comment by jengalas** [comment]
Posted by Zhanna
February 7, 2015 at 9:33 am
_Imported from WordPress_

Steve, thank you for your contribution! This is a really interesting investigation. It sure sounds to me like you’re correct that the NGS description points to Beacon 52 having been at Brent Point and not where the NGS coordinates specify.

I don’t know what to suggest about the coordinates that are, or appear to be, incorrect. I don’t see any indication of a simple transposition of numbers, which was my first hypothesis. Perhaps it was simply an error made on site or a data entry error (I’ve seen some benchmarks listed in the wrong county or quad before), or a mixup with another beacon that was at the listed coordinates (that we have yet to identify)? I don’t know why the topo map would show a beacon in that location if nothing else does, unless perhaps the mapmakers were using the NGS data.

I also considered that the beacon might in fact be part of a different airway, but with the limited NGS and airway map data available, I haven’t been able to find an airway that would fit. I searched the area for a “Beacon Light Road” or similar-named road nearby, which I see surprisingly often and can be a clue to former beacon locations—but no luck this time. It might also be worth reading the descriptions of nearby survey marks to see if any of them mention a beacon.

Is this near enough to you that you would be able to do a field search at some point?

**Reply by jengalas** [reply]
Posted by Steve Bennett
June 4, 2015 at 8:19 pm
_Imported from WordPress_

Zhanna,

Thanks for posting my findings.
I will report back when I get a chance to search the location on foot.
-Steve
